frontend/src/metabase/static-viz/components/XYChart/XYChart.tsx

```diff
@@ -310,6 +200,100 @@ export const XYChart = ({
         lineHeight={style.legend.lineHeight}
         fontSize={style.legend.fontSize}
       />
+
+      <Group left={valuesLeftOffset}>
```

Comment:
Group both the bottom axis and the chart together so I could apply a single left transform here.